app/utils/gdd.py

```diff
@@ -80,6 +86,54 @@ def calculate_gdd(db: Session, parcel: Parcel, disease_models: List[Disease],
             gdd_values=gdd_values
         )
 
-        disease_model_list.append(response_obj)
+        has_member_list = []
+
+        for gdv in gdd_values:
+            some_uuid = uuid4()
+            has_member_list.append(
+                {
+                    "@id": "urn:openagri:accumulatedGDD1:obs5:{}".format(some_uuid),
+                    "@type": "Observation",
+                    "phenomenonTime": "{}".format(str(gdv.date)),
+                    "hasResult": {
+                        "@id": "urn:openagri:accumulatedGGD1:obs4:result:{}".format(some_uuid),
+                        "@type": "QuantityValue",
+                        "numericValue": "{}".format(gdv.accumulated_gdd),
```

Comment:
I think the API has changed and instead `hasValue` is used in the place of `numericValue`
@arruda is that correct?